ginkgo.Jenkinsfile

```diff
@@ -54,66 +53,173 @@ pipeline {
                }
             }
         }
-        stage('Boot VMs'){
+        stage ("copy-code-and-boot-vms"){
```

Comment:
It seems like we can include spaces and capital letters here, am I missing some reason to avoid them? (like filenames or something?)

Reply:
Will do